src/other/rjsf/InputWithInfoPopover.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import Typography from "@mui/material/Typography";
+import { Templates } from "@rjsf/mui";
+import { WidgetProps } from "@rjsf/utils";
+import React from "react";
+
+import InfoPopover, {
+    InfoPopoverProps,
+} from "../../mui/components/popover/info-popover/InfoPopover";
+import { PositionInfoPopover } from "./PositionInfoPopover.styled";
+
+const { BaseInputTemplate } = Templates;
+
+type InfoPopoverOptions = InfoPopoverProps & { content: string };
+
+export default function InputWithInfoPopover(props: WidgetProps) {
```

Comment:
I wonder if we should mark somewhere what role the components fulfill (e.g. Template, Widget, or Field). If we plan to add a couple of more RJSF components, then perhaps we can divide them into subfolders named templates, widgets, and fields. As we have only 4 right now, perhaps a simple JSDoc string would suffice. What do you think?

Reply:
Agree, dividing into subfolders make a clean separation.